google/cloud/storage/hashing_options.h

```diff
@@ -45,10 +46,20 @@ struct MD5HashValue
 };
 
 /**
- * Compute the MD5 Hash of a string in the format preferred by GCS.
+ * Compute the MD5 Hash of a buffer in the format preferred by GCS.
  */
+std::string ComputeMD5Hash(absl::string_view payload);
+
+/// @copydoc ComputeMD5Hash(absl::string_view)
 std::string ComputeMD5Hash(std::string const& payload);
 
+/// @copydoc ComputeMD5Hash(absl::string_view)
+inline std::string ComputeMD5Hash(char const* payload) {
+  auto p =
+      payload == nullptr ? absl::string_view{} : absl::string_view{payload};
+  return ComputeMD5Hash(std::move(p));
```

Comment:
Moving a `string_view` is an anti-pattern, I believe (et seq).

Reply:
Fixed (everywhere I hope).